src/Sulu/Component/Websocket/ConnectionContext/ConnectionContext.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Websocket\ConnectionContext;
+
+use Guzzle\Http\Message\RequestInterface;
+use Guzzle\Http\QueryString;
+use Ratchet\ConnectionInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+/**
+ * Simple websocket context
+ */
+class ConnectionContext implements ConnectionContextInterface
+{
+    /**
+     * Return id of connection context for connection
+     * @param ConnectionInterface $conn
+     * @return string
+     */
+    public static function getIdFromConnection(ConnectionInterface $conn)
+    {
+        $dummy = new ConnectionContext($conn);
```

Comment:
i have to create a new instance to check the id of the connection because i dont want to copy the code from the constructur! i will rename the var to context ...